Tribler/Test/test_multichain_startup.py

```diff
@@ -0,0 +1,47 @@
+"""
+This file test whether Tribler is able to start when multichain is enabled
+Based on test_bartercast4_community.py
+"""
+from Tribler.Test.test_as_server import TestAsServer
+from Tribler.community.multichain.community import MultiChainCommunity, MultiChainCommunityCrawler
+from Tribler.community.tunnel.tunnel_community import TunnelSettings
+
+from Tribler.dispersy.util import blocking_call_on_reactor_thread
+
+from time import sleep
+
+
+class TestMultichainStartup(TestAsServer):
+
+    """Start a Tribler session and ensure it does not crash."""
+
+    def __init__(self, *argv, **kwargs):
+        super(TestMultichainStartup, self).__init__(*argv, **kwargs)
+
+    def test_multichain_startup(self):
+        sleep(5)
```

Comment:
This file loads the multichain community with the Tribler core. It will only catch pretty obvious bugs, but this does actually increase coverage.